multinode/bootstrapping.md

```diff
@@ -0,0 +1,191 @@
+it is properly setup to work as a data node. This process is referred
+to as *bootstrapping the data node*, which is done automatically each
+time a data node is added to an access node (unless `bootstrap` is
+false, in which case it just verifies that the data node is properly
+set up).
+
+The steps performed when boostrapping a data node are:
+
+1. Connect to the data node and authenticate the user.
+1. Check that the database is on the data node and create it
+   otherwise.
+2. Check that the TimescaleDB extension is available in the database or
+   create it otherwise.
+3. Set metadata on the data node.
+
+## Authentication during bootstrapping
+
+All connections to the data node are made as the current user on the
+access node. Which requires that:
+
+1. The current user exists on both the data node and access node.
+2. The data node is set up to handle logins using either certificates
+   or password files, as described in the [Data Node Authentication
+   Chapter][distdb-auth].
+
+## Adding data nodes to the access node
+
+A new data node is added to the access node using the
+[`add_data_node`][add_data_node] command.
+
+```sql
+SELECT * FROM add_data_node('data_node_1', 'data1.example.com');
+```
+
+The access node has to ensure that the database on the data node is
+properly configured by either bootstrapping the database or verifying
+that the database exists and is correctly configured. This is
+controlled with the `bootstrap` option, which can be either `true`
+(indicating that the data node should be bootstrapped) or `false`
+(indicating that the data node will only be validated, but no
+bootstrapping will take place).
+
+```sql
+SELECT * FROM add_data_node('data_node_1', 'data1.example.com',
+                            boostrap => false);
+```
+
+The default value for the `bootstrap` option is `true`, which means
+that the data node is bootstrapped when added.
+
+## Setting up the database on the data node
+
+The database to bootstrap on the data node will have the name of the
+current database on the access node by default. For example, if you
+are connected to the `postgres` database on the access node, the
+database name will be `postgres` on the data node as well.
+
+To provision the data node in a different database on the instance use
+the `database` option to the [`add_data_node`][add_data_node] command,
+for example:
+
+```sql
+SELECT * FROM add_data_node('data_node_1', 'data1.example.com',
+                            database => 'my_database');
+```
+
+This is particularly useful if you want to experiment with having data
+nodes and access nodes in the same Postgres instance.
+
+### Bootstrapping the database
+
+If `bootstrap` is `true` (which is the default), the access node will
+first connect to either the `postgres` database or, if the `postgres`
+database does not exist, to the `template0` database on the data node,
+and create a new database:
+
+```sql
+CREATE DATABASE <i>database</i>
```

Comment:
These \<i\> element tags are not functioning how you anticipated?
![Screen Shot 2020-01-28 at 2 24 16 PM](https://user-images.githubusercontent.com/12288273/73297678-fe659400-41d9-11ea-9576-55c963782d86.png)

Better to just put a space between '\<' and the text, i.e. '< database >'

Reply:
Ok. Updated.